.github/renovate.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "packageRules": [
+    {
+      "matchUpdateTypes": ["major", "minor"],
+      "enabled": false
```

Comment:
Just to be sure I understood correctly, this means do not try to update major or minor version, only patch version, right?

Reply:
Yes, major and minor version must be kept same for same Alpine ROS image tag